src/DataExporter.ts

```diff
@@ -1,14 +1,18 @@
+import * as fs from "fs";
 /*---------------------------------------------------------------------------------------------
 * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
 * See LICENSE.md in the project root for license terms and full copyright notice.
 *--------------------------------------------------------------------------------------------*/
-import { DbResult, Id64Array, Logger, LogLevel } from "@bentley/bentleyjs-core";
-import { BackendRequestContext, ECSqlStatement, IModelDb } from "@bentley/imodeljs-backend";
-import { MassPropertiesOperation, MassPropertiesRequestProps } from "@bentley/imodeljs-common";
 import * as path from "path";
-import * as fs from "fs";
 
-const loggerCategory = "DataExporter";
+import { ECSqlStatement } from "@itwin/core-backend/lib/cjs/ECSqlStatement";
+import { IModelDb } from "@itwin/core-backend/lib/cjs/IModelDb";
+import { DbResult } from "@itwin/core-bentley/lib/cjs/BeSQLite";
+import { Id64Array } from "@itwin/core-bentley/lib/cjs/Id";
+import { Logger, LogLevel } from "@itwin/core-bentley/lib/cjs/Logger";
+import { MassPropertiesOperation, MassPropertiesRequestProps } from "@itwin/core-common/lib/cjs/MassProperties";
```

Comment:
Import all of the dependencies from the bundle (i.e. @itwin/core-bentley or other package) rather than the individual file